addons/l10n_nl/migrations/3.3/post-migrate_update_taxes.py

```diff
@@ -0,0 +1,81 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+from odoo import api, SUPERUSER_ID
+from psycopg2 import sql
+
+
+def _get_tax_ids_for_xml_id(cr, xml_id):
+    xml_id = '%' + xml_id
+    cr.execute(sql.SQL(
+        """
+        SELECT res_id
+        FROM ir_model_data
+        WHERE model = 'account.tax'
+        AND name ILIKE {xml_id}
+        """
+    ).format(xml_id=sql.Literal(xml_id)))
```

Comment:
```suggestion
    cr.execute(sql.SQL(
        """
        SELECT res_id
        FROM ir_model_data
        WHERE model = 'account.tax'
        AND name LIKE %{xml_id}
        """
    ).format(xml_id=sql.Literal(xml_id)))
```
XML ids are case sensitive :wink: